lib/simctl.js

```diff
@@ -732,6 +732,57 @@ async function getDeviceTypes () {
   }
 }
 
+
+/**
+ * Get the full list of runtimes, devicetypes, devices and pairs in JSON
+ *
+ * @return {Array<string>} List of the device types, runtimes devices and pairs JSO
```

Comment:
JSON?

Reply:
I alos think the return value should be `{Object}`